### DISCUSSION TITLE
Unable to join the students discount.

### DISCUSSION THREAD

**Discussion by snyper-coder** [community/community#50775] [GitHub Education]
I'm trying to apply for the student discount but it keeps rejecting My college Kamla Nehru Institute of Physical & Social Sciences doesn't provide email ids they have mentioned my personal Gmail id on the college id card. In the first attempt, my application was rejected as I wasn't aware that your profile should be the same as on your id. After fixing that I tried again and it wasn't accepting my college id saying that proof has been rejected already so I tried uploading copy of the fee receipt (dated receipt from the bursar) and it got rejected again asking me to use my college id. I believe my id got rejected because the institute only mentions the batch on the id, not an expiry date. But that's the only id I got :( dono what to do now, I hv my mid sem hall ticket as well but I doubt it will work either. 
![image](https://user-images.githubusercontent.com/62513313/227103663-70f47da2-9898-43f1-b885-ec8e0c03b9a3.png)

**Comment by samus-aran** [comment]
Hi @snyper-coder 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

We also ask you to re-submit a different image as it has been used previously. We do allow multiple pieces of evidence in the same photo if possible.

Hope this helps! Let us know how you get along 👍

**Comment by KHOANGUYEN196197** [comment]
.
